model/schema/core.yaml

```diff
@@ -355,71 +356,74 @@ classes:
       - prefix_order
 
   Chromosome:
-    is_a: AuditedObject
-    description: The ID of the landmark used to establish the coordinate system for the current feature.
+    is_a: GenomicEntity
+    description: >-
+      A structure composed of a single DNA molecule and associated proteins
+      that carries here heriditary information from cell to cell. In the context
+      of this model, it is used as a landmark for cytogenetic and genetic
+      mapping of genes, alleles and chromosomal aberrations; this entity ties
+      together related chromosome sequences from different genome assemblies.
+    notes: >-
+      I'm trying to distinguish the idealized biological entity from a species'
+      genome from a chromosome sequence assembly that is specific to a given
+      genome assembly version.
+    exact_mappings:
+      - SO:0000340
+      - GO:0005694
+
+  ChromosomeAssembly:
+    is_a: GenomicEntity
+    description: >-
+      A pseudo-molecule assembled from smaller sequences, representing most or
+      all of a chromosome's DNA sequence. In the context of this model, this
+      entity is used as a landmark to establish a coordinate system for the
+      location of other genomic entities.
     slots:
-      - curie
       - display_name
-    slot_usage:
-      curie:
-        required: true
+      - has_assembly
```

Comment:
I'd prefer this to just be called `assembly` - `has_assembly` sounds like the name of a boolean field to me

Reply:
I'll change this to `genome_assembly`.

Reply:
yeah, I forgot slot names can't be the same as class names - `genome_assembly` sounds good